spec/system/partner_system_spec.rb

```diff
@@ -532,6 +532,39 @@
           assert page.has_content? item_category_2.name
         end
       end
+
+      describe 'deleting an existing partner group' do
```

Comment:
This can be a request spec rather than a system spec (which is heavier and flakier).

Reply:
Sure will replace it the feature test.